src/plopp/backends/matplotlib/utils.py

```diff
@@ -27,14 +27,19 @@ def fig_to_bytes(fig: Figure, form: Literal['png', 'svg'] = 'png') -> bytes:
 
 def is_interactive_backend() -> bool:
     """
-    Return ``True`` if the current backend used by Matplotlib is the widget/ipympl
-    backend.
+    Return ``True`` if the current backend used by Matplotlib creates interactive
+    figures. See
+    https://matplotlib.org/stable/users/explain/figure/backends.html#the-builtin-backends
+    for a list of backends.
     """
-    backend = mpl.get_backend()
-    return any(x in backend for x in ("ipympl", "widget"))
+    backend = mpl.get_backend().lower()
+    return any(
+        b in backend
+        for b in ('qt', 'ipympl', 'gtk', 'tk', 'wx', 'nbagg', 'web', 'macosx', 'widget')
```

Comment:
Not sure if this is any prettier but this can be accessed via
```
matplotlib.backends.backend_registry.list_builtin(matplotlib.backends.BackendFilter.INTERACTIVE)
```

Reply:
The widget one is missing from the list.

Reply:
Thanks I was looking for something like that, although it does not include the `ipympl` backend...

Reply:
Apparently, the `backend_registry` does not exist in older versions of `matplotlib` (at least it's not there in 3.7.3).
So I'm just going to stick with my custom list for now.